### PR TITLE
fix: switch component

### PR DESCRIPTION
**Pull request by Highjhacker**
## Summary

Fix the `<x-ark-switch>` component that wasn't visible before (related to Nodem)

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [x] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: resources/views/inputs/switch.blade.php [hunk 1]
```diff
@@ -2,28 +2,32 @@
     x-data="{ value: {{ $default ?? 'false' }}, toggle() { this.value = !this.value; this.$refs['checkbox-livewire'].click(); }, focused: false }"
     class="flex items-center space-x-3"
 >
-    <span class="font-semibold {{ $labelClass ?? '' }}" :class="{ 'text-theme-secondary-500': !value }">
+    <span class="font-semibold {{ $labelClass ?? '' }}" :class="{ 'text-theme-secondary-500': value }">
```
Comment (ItsANameToo) [inline resources/views/inputs/switch.blade.php:5]: why did this change?

File: resources/views/inputs/switch.blade.php [hunk 2]
```diff
@@ -35,7 +39,7 @@ class="hidden"
         @if($default ?? false) checked @endif
     />
 
-    <span class="font-semibold {{ $labelClass ?? '' }}" :class="{ 'text-theme-secondary-500': value }">
+    <span class="font-semibold {{ $labelClass ?? '' }}" :class="{ 'text-theme-secondary-500': !value }">
```
Comment (ItsANameToo) [inline resources/views/inputs/switch.blade.php:42]: why did this change?

File: resources/views/inputs/switch.blade.php [hunk 3]
```diff
@@ -2,28 +2,32 @@
...
-        <span aria-hidden="true" class="input-toggle-slide"></span>
+        <span
+            aria-hidden="true"
+            class="absolute mx-auto w-full h-1.5 rounded-full transition-colors duration-200 ease-in-out bg-theme-secondary-300 dark:bg-theme-secondary-800"
+        ></span>
         <span
             aria-hidden="true"
             :class="{
                 'input-switch-button-left': value,
                 'input-switch-button-right': !value
```
Comment (ItsANameToo) [inline resources/views/inputs/switch.blade.php:27]: why did this not change when some other values were inverted?

File: resources/views/inputs/switch.blade.php (hunk 1 above)
Comment (Highjhacker) [reply resources/views/inputs/switch.blade.php:5]: No clues of how it worked before, but was just the wrong direction when I tried it, so changed.

File: resources/views/inputs/switch.blade.php (hunk 2 above)
Comment (Highjhacker) [reply resources/views/inputs/switch.blade.php:42]: No clues of how it worked before, but was just the wrong direction when I tried it, so changed.

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**